core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -1556,6 +1681,133 @@ public void warmUp() {
         }
     }
 
+    private void checkDetachedMetadata(long timestamp) {
+        // load/check _meta
+        other2.of(detachedPath).concat(META_FILE_NAME).$();
+        if (!ff.exists(other2)) {
+            // Backups and older versions of detached partitions will not have _meta
+            LOG.info().$("detached _meta not found, skipping check [path=").$(other2).I$();
+            return;
+        }
+        try {
+            detachedMetadata.deferredInit(other2, ColumnType.VERSION);
+            if (metadata.getId() != detachedMetadata.getId()) {
```

Comment:
Check that designated timestamp columns are the same

Reply:
yes, done.